raiden/blockchain/events.py

```diff
@@ -249,6 +250,11 @@ def decode_raiden_event_to_internal(
     data["transaction_hash"] = log_event["transactionHash"]
     data["block_hash"] = bytes(log_event["blockHash"])
 
+    # Remove the old names
+    del data["blockNumber"]
+    del data["transactionHash"]
+    del data["blockHash"]
```

Comment:
I either would have to format this to hex or remove them, since don't use it internally I decided to go with removing the extra keys.